airflow-core/src/airflow/ui/src/theme.ts

```diff
@@ -34,140 +34,336 @@ const customConfig = defineConfig({
+          "300": { value: "oklch(86.9% .005 56.366)"},
+          "400": { value: "oklch(70.9% .01 56.259)"},
+          "500": { value: "oklch(55.3% .013 58.071)"},
+          "600": { value: "oklch(44.4% .011 73.639)"},
+          "700": { value: "oklch(37.4% .01 67.558)"},
+          "800": { value: "oklch(26.8% .007 34.298)"},
+          "900": { value: "oklch(21.6% .006 56.043)"},
+          "950": { value: "oklch(14.7% .004 49.25)"},
         },
       },
     },
     semanticTokens: {
       colors: {
-        success: generateSemanticTokens("success"),
-        failed: defaultConfig.theme?.semanticTokens?.colors?.red ?? {},
-        queued: generateSemanticTokens("queued"),
-        skipped: defaultConfig.theme?.semanticTokens?.colors?.pink ?? {},
-        up_for_reschedule: defaultConfig.theme?.semanticTokens?.colors?.cyan ?? {},
-        up_for_retry: defaultConfig.theme?.semanticTokens?.colors?.yellow ?? {},
-        upstream_failed: defaultConfig.theme?.semanticTokens?.colors?.orange ?? {},
-        running: generateSemanticTokens("running"),
-        restarting: generateSemanticTokens("restarting"),
-        deferred: generateSemanticTokens("deferred"),
-        scheduled: generateSemanticTokens("scheduled"),
-        none: generateSemanticTokens("none", "black"),
-        removed: generateSemanticTokens("removed", "black"),
+        navbar: generateSemanticTokens("gray"),
+        // TAILWIND 4.0 COLORS
+        red: generateSemanticTokens("red"),
+        orange: generateSemanticTokens("orange"),
+        amber: generateSemanticTokens("amber"),
+        yellow: generateSemanticTokens("yellow"),
+        lime: generateSemanticTokens("lime"),
+        green: generateSemanticTokens("green"),
+        emerald: generateSemanticTokens("emerald"),
+        teal: generateSemanticTokens("teal"),
+        cyan: generateSemanticTokens("cyan"),
+        sky: generateSemanticTokens("sky"),
+        blue: generateSemanticTokens("blue"),
+        indigo: generateSemanticTokens("indigo"),
+        violet: generateSemanticTokens("violet"),
+        purple: generateSemanticTokens("purple"),
+        fuchsia: generateSemanticTokens("fuchsia"),
+        pink: generateSemanticTokens("pink"),
+        rose: generateSemanticTokens("rose"),
+        slate: generateSemanticTokens("slate"),
+        gray: generateSemanticTokens("gray"),
+        zinc: generateSemanticTokens("zinc"),
+        neutral: generateSemanticTokens("neutral"),
+        stone: generateSemanticTokens("stone"),
+        // AIRFLOW TASK STATE
+        success: generateSemanticTokens("green"),
+        failed: generateSemanticTokens("red"),
+        queued: generateSemanticTokens("stone"),
+        skipped: generateSemanticTokens("pink"),
+        up_for_reschedule: generateSemanticTokens("cyan"),
+        up_for_retry: generateSemanticTokens("yellow"),
+        upstream_failed: generateSemanticTokens("orange"),
+        running: generateSemanticTokens("emerald"),
```

Comment:
emerald is too similar to green. We should have strong differentiation.

Reply:
I've changed the "running" to be cyan instead and updated the screenshots.
I can always switch back to "emerald" and make "lime" more distinct, too if you prefer.